A32-xds-circuit-breaking.md

```diff
@@ -55,11 +55,13 @@ limits to very high values, for example, to
 
 ## Detailed design
 
+### Threshold enforcement in LB policy
+
 With the introduction of [gRFC A28: xDS Traffic Splitting and Routing](https://github.com/grpc/proposal/blob/master/A28-xds-traffic-splitting-and-routing.md), 
-it became possible for there to be more than one CDS LB policy instance for a 
-given cluster (e.g., if one route pointed to a given cluster and a different 
-route split traffic between that cluster and other clusters).  However, after 
-implementing the changes described in [gRPC A31: xDS RouteActions Support](https://github.com/grpc/proposal/pull/192),
+it became possible for each client channel to have more than one CDS LB policy 
+instances for a given cluster (e.g., if one route pointed to a given cluster
```

Comment:
s/instances/instance/

Reply:
Well, shouldn't the noun following "more than one" be plural? I don't find an authoritative answer. Anyway, I am trusting you :)